modules/tinymce/src/core/main/ts/dom/ElementUtils.ts

```diff
@@ -111,8 +122,12 @@ const ElementUtils = (dom: DOMUtils): ElementUtils => {
     return !Bookmarks.isBookmarkNode(node1) && !Bookmarks.isBookmarkNode(node2);
   };
 
+  const isAttributeInternal = (attributeName: string): boolean =>
+    Arr.find(internalAttributesPrefixes, (value) => Strings.startsWith(attributeName, value)).isSome() || internalAttributes.has(attributeName);
```

Comment:
```suggestion
    Arr.exists(internalAttributesPrefixes, (value) => Strings.startsWith(attributeName, value)) || internalAttributes.has(attributeName);
```
Sorry had missed this previously, but `exists` is the same as using `find` with `isSome()`